src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs

```diff
@@ -229,6 +229,7 @@ void RegisterEvents()
 			if (_textBox != null)
 			{
 				_textBox.KeyDown += OnTextBoxKeyDown;
+				_textBox.LostFocus += OnTextBoxLostFocus;
```

Comment:
I think this solution may be problematic in case the user wants to use keyboard to navigate the suggestions - I would think that in such case the `TextBox` will lose focus and that will then cause the suggestion list to close, is that not the case? Maybe focus navigation is not yet correctly implemented in Uno for `AutoSuggestBox`

Reply:
Ah right, probably not - seems the WinUI control keeps focus on the input when keys are pressed and Tab actually causes the suggestion list to close 👍